src/Nethermind/Nethermind.Core/StorageCell.cs

```diff
@@ -3,27 +3,46 @@
 
 using System;
 using System.Diagnostics;
+using Nethermind.Core.Crypto;
 using Nethermind.Int256;
 
 namespace Nethermind.Core
 {
     [DebuggerDisplay("{Address}->{Index}")]
     public readonly struct StorageCell : IEquatable<StorageCell>
     {
+        private readonly ValueHash256? _hash;
```

Comment:
Is this increasing the size of every `StorageCell` by 50%?

Reply:
optimized